x-pack/legacy/plugins/xpack_main/server/lib/setup_xpack_main.js

```diff
@@ -22,8 +22,7 @@ export function setupXPackMain(server) {
 
   server.ext('onPreResponse', (request, h) => injectXPackInfoSignature(info, request, h));
 
-  const { registerFeature, getFeatures } = server.newPlatform.setup.plugins.features;
-  server.expose('registerFeature', registerFeature);
+  const { getFeatures } = server.newPlatform.setup.plugins.features;
   server.expose('getFeatures', getFeatures);
```

Comment:
Are 3rd party plugins meant to be used with a licensed version of Kibana supposed to be able to use x-pack features/apis?

If so, should we add a dev doc or anything to warn that `registerFeature` is no longer usable from legacy plugins?

Reply:
ok, I will do. just in case